ios/Classes/SwiftFlutterLoginFacebookPlugin.swift

```diff
@@ -5,7 +5,7 @@ import FBSDKLoginKit
 
 /// Plugin methods.
 enum PluginMethod: String {
-    case logIn, logOut, getAccessToken, getUserProfile, getUserEmail, getSdkVersion, getProfileImageUrl
+    case logIn, logOut, expressLogin, getAccessToken, getUserProfile, getUserEmail, getSdkVersion, getProfileImageUrl
```

Comment:
I think we shouldn't add this method in iOS implementation, since it's only available for Android. It's confusing